server/game/cards/01-Core/TheSting.js

```diff
@@ -6,9 +6,13 @@ class TheSting extends Card {
             effect: ability.effects.skipStep('key')
         });
 
-        this.persistentEffect({
-            targetController: 'opponent',
-            effect: ability.effects.forgeAmberGainedByOpponent()
```

Comment:
Can you cleanup this effect "forgeAmberGainedByOpponent" from effect.js?

Reply:
Done. Tests passed as well.